proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -90,6 +91,18 @@ public VelocityConfiguration getConfiguration() {
         return configuration;
     }
 
+    @Override
+    public void reloadConfiguration() {
+        // TODO: reload configuration
+
+        // Give a chance for plugins to reload as well
+        try {
+            eventManager.fire(new ProxyReloadEvent()).get();
```

Comment:
Not entirely sure if this should be blocking

Reply:
It shouldn't be. Use `fireAndForget()`.